system-tests/__snapshots__/cdp_spec.ts.js

```diff
@@ -19,44 +19,10 @@ exports['e2e cdp / handles disconnections as expected'] = `
 
   e2e remote debugging disconnect
     ✓ reconnects as expected
-
 There was an error reconnecting to the Chrome DevTools protocol. Please restart the browser.
```

Comment:
This change is due to how we are now treating error reconnecting to the Chrome DevTools as a fatal failure:

https://github.com/cypress-io/cypress/pull/19915/files#diff-4eee46e0819a421b758964abfcf2ea357773afceaff00ceafd16b89f193f4c4eR190-R194